frontend/src/Explorer/View/Footer.purs

```diff
@@ -143,7 +143,7 @@ navItemsRow2 lang =
       , link: "https://tada.iohk.io/"
       }
     , { label: translate (I18nL.footer <<< I18nL.fooCardanoSLDocumentation) lang
-      , link: "https://cardano-docs.iohk.io/introduction/"
+      , link: "https://cardanodocs.com/introduction/"
```

Comment:
ps, there is no need to hardcode url of the page being shown. You can find out hostname easily with https://github.com/input-output-hk/cardano-sl-explorer/blob/master/frontend/src/Explorer/Util/Config.purs#L26